physical/etcd3.go

```diff
@@ -148,13 +149,16 @@ func (c *Etcd3Backend) Get(key string) (*Entry, error) {
 	if len(resp.Kvs) == 0 {
 		return nil, nil
 	}
+	if len(resp.Kvs) > 1 {
+		panic("unexpected number of keys from a get request")
```

Comment:
Please don't panic. An error here will be sufficient to log an issue and fail the client request.